torch/_inductor/codegen/wrapper.py

```diff
@@ -1058,6 +1058,12 @@ def define_kernel(
     def generate_return(self, output_refs):
         # Output tensors are allocated by the AOT runtime.
         if V.graph.aot_mode:
+            for idx, output in enumerate(V.graph.graph_outputs):
+                if output.get_name() in self.reuses:
```

Comment:
It seems better to just make `can_reuse()` return False for outputs rather than adding a copy.

Reply:
Ok, I gave that a try and it revealed a subtle issue. Because we take a two-pass codegen, in the first pass we generate the python code the reuse will happen (because there is no passed-in output tensors). In the second pass, `can_reuse()` returns false, and then the generated triton kernel will be different (with different number of args).

I can fix that by introducing some state so that when WrapperCodeGen generates code, it knows it's in an aot mode, but I am afraid that will make more and more aot logic bleed into the base WrapperCodeGen. For the time being, can we keep the `copy_` here?

Reply:
I'll approve to unblock, but I think we should figure out how not to do a copy here.  The copy will slow down AotAutograd.